apps/src/StudioApp.js

```diff
@@ -858,20 +853,20 @@ StudioApp.prototype.showInstructions_ = function(level, autoClose) {
 
   if (autoClose) {
     setTimeout(function() {
-      dialog.hide();
+      this.instructionsDialog.hide();
     }, 32000);
```

Comment:
Function needs to be bound to `this` or `this.instructionsDialog` will be undefined.